src/librustc/session/config.rs

```diff
@@ -1401,6 +1401,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "run the self profiler"),
     profile_json: bool = (false, parse_bool, [UNTRACKED],
         "output a json file with profiler results"),
+    dump_profiler_events: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
This option isn't used and the other profiler related options should be cleaned up.